libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -367,9 +367,20 @@ def prepare_output_stream(
             if provider == "cohere" and (
                 chunk_obj["is_finished"] or chunk_obj[output_key] == "<EOS_TOKEN>"
             ):
+                yield _get_invocation_metrics_chunk(chunk_obj)
```

Comment:
This is an optional change, if you believe there is risk in adding it, happy to remove it. It is not the core focus of the review.

I couldn't find a reason why metrics weren't included for cohere.

Reply:
Done. Removed in latest push.